static/admin/config.yml

```diff
@@ -183,4 +183,60 @@ collections:
               - label: "Color Description"
                 name: "description"
                 widget: "string"
+      - {label: "Version No.", name: "version", widget: "select", options: ["1.01", "1.02", "1.03","1.04", "1.05"]}
+  - name: "typography"
+    label: "Typography"
+    folder: "src/pages/text"
+    create: true
+    extension: "json"
+    format: "json"
+    editor:
+     preview: false
+    slug: "{{fields.version}}-{{slug}}"
```

Comment:
@ruaraikirk can you update to `    slug: "{{fields.version}}-{{fields-srcTemplateKey}}"`?